rendering/mmd_pmd.ksy

```diff
@@ -0,0 +1,368 @@
+meta:
+  id: mmd_pmd
+  title: MMD model data (older format)
```

Comment:
I guess here it shouldn't be abbreviated. Also please put each format into an own PR.

Reply:
I'll just put in both like `MMD (MikuMikuDance)` since MMD is what people are going to be using as their search term.

As for splitting the PR... ugh. Yeah, sure. Once I'm done with the first round of changes I'll split it and resubmit.